### PR TITLE
Task#2

### PR DESCRIPTION
**Pull request by TarekMGamal**

### CODE REVIEW

File: albums/models.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from artists.models import Artist
+
+class Album(models.Model):
+    name = models.CharField(max_length=100, default='New Album')
+    creation_datetime = models.DateTimeField(auto_now_add=True)
+    release_datetime = models.DateTimeField(blank=False)
+    cost = models.DecimalField(blank=False, max_digits=9, decimal_places=2)
+    artist = models.ForeignKey(Artist, on_delete=models.CASCADE)
+    isApproved = models.BooleanField(default=False)
```
Comment (AhmedMorgan) [inline albums/models.py]:
```suggestion
    is_approved = models.BooleanField(default=False)
```

File: albums/models.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from artists.models import Artist
+
+class Album(models.Model):
+    name = models.CharField(max_length=100, default='New Album')
+    creation_datetime = models.DateTimeField(auto_now_add=True)
```
Comment (AhmedMorgan) [inline albums/models.py]: you should make this field not editable

File: albums/models.py (hunk 1 above)
Comment (TarekMGamal) [reply albums/models.py]: Done

File: albums/models.py (hunk 2 above)
Comment (TarekMGamal) [reply albums/models.py]: Done

**Pull request merged**